### PR TITLE
MWPW-153237: Test content for CCD validation

### PR DESCRIPTION
**Pull request by 3ch023**
Add a gallery page with CCD card variations.

Cards are pulled from Odin. At the moment you need VPN to access Odin instance.
Includes Dark and Light mode.
Known Missing Items (due to feature incomplete of Studio/Cards):

- strikethrough price
- yellow badge
- card without a mnemonic icon
- card with arbitrary CTA text (e.g. 'Try '
- Suggested Card - background image (aka border image)

Resolves: https://jira.corp.adobe.com/browse/MWPW-153237

Access the gallery at https://ccd--milo--adobecom.hlx.page/libs/features/mas/docs/ccd.html?theme=dark
CCD branch is required due to dependencies on other PRs. 

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/?martech=off
- After: https://ccd-gallery--milo--adobecom.hlx.page/?martech=off

### CODE REVIEW

**Review by yesil — APPROVED** [review]

File: libs/features/mas/docs/ccd.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    <link rel="stylesheet" href="../../../styles/styles.css">
+    <!-- Include any additional stylesheets -->
+    <link rel="stylesheet" href="styles.css">
+  </head>
+
+  <body>
+    <main>
+      <sp-theme color="light" scale="medium"> 
+        <div class="gallery-content">
+          <div class="vpn-banner">
```
[the remark this answers is not in the document]
Comment (3ch023) [reply libs/features/mas/docs/ccd.html:46]: i know but for a week it didn't seem worth the effort

File: libs/features/mas/docs/styles.css [hunk 2]
```diff
@@ -105,6 +105,37 @@ pre {
   background-color: var(--spectrum-global-color-gray-100);
 }
 
-#log {
+#log,
+#log2,
+#log3 {
```
[the remark this answers is not in the document]
Comment (3ch023) [reply libs/features/mas/docs/styles.css:110]: ho, true. copy paste from ccd state..

File: libs/features/spectrum-web-components/build.mjs [hunk 3]
```diff
@@ -97,8 +99,7 @@ build({
 });
 
 mods.forEach((mod) => {
-  if (mod === 'lit.js') return;
-  if (mod === 'polyfills') return;
+  if (mod === 'lit.js' || mod === 'polyfills' || mod === 'themes') return;
```
[the remark this answers is not in the document]
Comment (3ch023) [reply libs/features/spectrum-web-components/build.mjs:102]: ah right, i remember i promised
will change to `['lit.js', 'polyfills', 'themes'].includes(mod)`, good?

File: libs/features/mas/docs/ccd.html [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+          
+          <h1 id="ccd-gallery" tabindex="-1">CCD Gallery <a class="header-anchor" href="#ccd-gallery" href="#ccd-gallery" title="Permalink to this heading">#</a></h1>
+          Switch Theme: <a href="?theme=dark">Dark</a> OR <a href="?theme=light">Light</a>
+
+          <h2 id="ccd-slice-card" tabindex="-1">CCD Slice Card <a class="header-anchor" href="#ccd-slice-card" href="#ccd-slice-card" title="Permalink to this heading">#</a></h2>
+          <div class="gallery-grid-3">
+            <h5>Percentage</h5>
+            <h5>Two icons & pricing</h5>
+            <h5>See Terms link</h5>
+            <merch-card><aem-fragment fragment="0ef2a804-e788-4959-abb8-b4d96a18b0ef"></aem-fragment></merch-card>
```
[the remark this answers is not in the document]
Comment (yesil) [reply libs/features/mas/docs/ccd.html:58]: It doesn't need to be matching with the exact title.
It is more descriptive, like cc, photoshop, illustrator.
I think as a best practice we should adopt it, maybe I'm wrong.

**Review by npeltier — APPROVED** [review]
lgtm

**Review by Axelcureno — APPROVED** [review]

**Pull request merged**